ng-dev/pr/common/validation/pull-request-failure.ts

```diff
@@ -10,16 +10,17 @@ import {breakingChangeLabel} from '../../config/index.js';
 import {TargetLabel} from '../targeting/target-label.js';
 
 /**
- * Class that can be used to describe pull request failures. A failure
- * is described through a human-readable message and a flag indicating
- * whether it is non-fatal or not.
+ * Class that can be used to describe pull request failures.
+ *
+ * A failure is described by a human-readable message and a flag indicating
+ * whether the failure can be forcibly ignore or not.
  */
 export class PullRequestFailure {
   constructor(
     /** Human-readable message for the failure */
     public message: string,
     /** Whether the failure is non-fatal and can be forcibly ignored. */
-    public nonFatal = false,
+    public canBeIgnoredForcibly = false,
```

Comment:
What about just `canBeIgnored`, `canBeIgnoredForcibly` doesn't quite sound right to me.  Another option might be `canBeIgnoredWithForced`?

Reply:
Changed to `canBeIgnoredNonFatal`